doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenario-designer-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-scenario-designer-con']
+= Using the Test Scenario (Legacy) designer in {CENTRAL}
+
+In {PRODUCT}, there are two test scenarios designers. One is the *Test Scenario* designer and the other is *Test Scenario (Legacy)* designer. You can use either of them to test your business rules & data. The *Test Scenario* designer is the default designer and you can create and run DMN-based test scenarios using it. This chapters covers how to use the *Test Scenario (Legacy)* designer to create and run test scenarios. The *Test Scenario (Legacy)* designer runs test scenario based on DRL files and Guided Decision Tables.
```

Comment:
Minor suggestion - using the more common word "describes" and pluralizing "scenarios":

s/covers/describes

s/The *Test Scenario (Legacy)* designer runs test scenario based on DRL files and Guided Decision Tables./The *Test Scenario (Legacy)* designer runs test scenarios based on DRL files and Guided Decision Tables.

Reply:
Changed as suggested.